templates/joomla/html/com_content/article/requirements.php

```diff
@@ -129,6 +129,11 @@
 					</td>
 				</tr>
 			</tbody>
+			<tbody>
```

Comment:
Pretty sure multiple tbody's are a bad thing :P If this is at the bottom would assume <tfoot> is more appropriate?

Reply:
When things are grouped the way they are it's fine.  We need to break this into two rows though; the current row for the header then another row that's the extra text.  Or, take it out of the table and just use an alert div or something after the table.